### PR TITLE
Added 'collisionrule' and 'deathmessagevisibility' options for teams & entity ai functions

### PR DESCRIPTION
**Pull request by macjuul**
The NameTagVisibility *should* also use this new Option system that the new team options use, although I have no idea how to make it backwards compatible right now, so I left it out of the change for now

### CODE REVIEW

File: src/main/java/com/laytonsmith/core/functions/Scoreboards.java [hunk 1]
```diff
@@ -412,11 +414,16 @@ public Construct exec(Target t, Environment environment,
 				if(Static.getServer().getMinecraftVersion().gte(MCVersion.MC1_8)) {
 					ops.set("nametagvisibility", new CString(team.getNameTagVisibility().name(), t), t);
 				}
+				if(Static.getServer().getMinecraftVersion().gte(MCVersion.MC1_9_4)) {
```
Comment (PseudoKnight) [inline src/main/java/com/laytonsmith/core/functions/Scoreboards.java]: This should be MCVersion.MC1_9, like in the other function.

File: src/main/java/com/laytonsmith/abstraction/enums/bukkit/BukkitMCOptionStatus.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	public static BukkitMCOptionStatus getConvertor() {
+		if (instance == null) {
+			instance = new BukkitMCOptionStatus();
+		}
+		return instance;
+	}
+
+	@Override
+	protected OptionStatus getConcreteEnumCustom(MCOptionStatus abstracted) {
+		if(Static.getServer().getMinecraftVersion().lt(MCVersion.MC1_9_4)) {
```
Comment (PseudoKnight) [inline src/main/java/com/laytonsmith/abstraction/enums/bukkit/BukkitMCOptionStatus.java]: Added in MC1_9. Check the others too. It was within a few days of the initial release. No one should be using first week builds, so 1_9 is fine.

File: src/main/java/com/laytonsmith/abstraction/bukkit/entities/BukkitMCEntity.java [hunk 3]
```diff
@@ -322,12 +323,17 @@ public MCLocation asyncGetLocation() {
...
+		if(ReflectionUtils.hasMethod(e.getClass(), "isGlowing", null)) { // Added around 1.9
+			return e.isGlowing();
+		}
+		return false;
 	}
 	
 	@Override
 	public void setGlowing(Boolean glow) {
-		e.setGlowing(glow);
+		if(ReflectionUtils.hasMethod(e.getClass(), "setGlowing", null)) { // Added around 1.9
```
Comment (PseudoKnight) [inline src/main/java/com/laytonsmith/abstraction/bukkit/entities/BukkitMCEntity.java]: These were added on day 1 of 1.9 builds, so no check is required here.

**Comment by PseudoKnight** [comment]
So, three version checks should be changed to 1_9, and the two method checks for glowing can be removed.

**Comment by macjuul** [comment]
I'll fix it asap.

Also, what is the best way to find out when something was added or changed?

**Comment by PseudoKnight** [comment]
I refer to Bukkit commit history here.
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/commits

**Comment by PseudoKnight** [comment]
This looks good to me.

**Pull request merged**